client/components/layout/client.coffee

```diff
@@ -22,42 +22,23 @@ module.exports.init = ->
   $.ajaxSettings.headers = 'X-Access-Token': sd.USER.access_token
   window[key] = helper for key, helper of viewHelpers
   Backbone.history.start pushState: true
-  initAnalyitcs()
+  @user = new User sd.USER
+  channelAutocomplete.init()
 
-# Replace broken profile icon
-imgLoad = imagesLoaded('#layout-sidebar-profile img')
-imgLoad.on 'fail', ->
-  $('#layout-sidebar-profile img').attr(
-    'src'
-    "/images/layout_missing_user.png"
-  )
+  # Replace broken profile icon
```

Comment:
This isn't needed anymore with the profile icon being a generic svg right?

![image](https://cloud.githubusercontent.com/assets/555859/16778431/fbe5bc8c-483b-11e6-9db7-27e5bc540b79.png)